import.py

```diff
@@ -651,6 +651,8 @@ def get_decompme_compiler_name(
 
     for path, compiler_name in compiler_mappings.items():
         assert isinstance(compiler_name, str)
+        # This is an annoying hack because decompme doesnt seem to know about macos
+        compiler_path = compiler_path.replace("macos", "linux")
```

Comment:
Hm... The original intent was that you would add another entry to your tools/permuter_settings.toml to handle this. Maybe we can add support for wildcards to that file?